packages/medusa/src/api/routes/admin/product-categories/index.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from "express"
+import middlewares from "../../../middlewares"
+import getProductCategory from './get-product-category'
+
+const route = Router()
+
+export default (app) => {
+  app.use("/product-categories", route)
```

Comment:
question: @olivermrbl should we wrap this in a feature flag until the feature is more complete in order to not release half an api next release cycle?

Reply:
Yeah, the thought struck me before, and I think it might be a good idea - even though the feature will be relatively isolated. Good push @pKorsholm :)

Reply:
added FF